docs/admin/cluster-troubleshooting.md

```diff
@@ -21,7 +21,7 @@ certainly want the docs that go with that version.</h1>
 
 <!-- END MUNGE: UNVERSIONED_WARNING -->
 # Cluster Troubleshooting
-Most of the time, if you encounter problems, it is your application that is having problems.  For application
+Most of the time, if you encounter problems, it is your application that is the root cause.  For application
```

Comment:
Wow, that's... bold of us. I would remove this or qualify with a "and your cluster has been set up correctly and passed the conformance test" or some such.